lib/node_modules/@stdlib/blas/base/dtrsm/README.md

```diff
@@ -0,0 +1,247 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dtrsm
+
+> Solve matrix equation `op(A) * X = alpha * B` or `X * op(A) = alpha * B` where `alpha` is a scalar, `X` and `B` are m by n matrices, `A` is a unit, or non-unit, upper or lower triangular matrix and `op( A )` is one of `op( A ) = A` or `op( A ) = A ** T`. The matrix `X` is overwritten on `B`.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var dtrsm = require( '@stdlib/blas/base/dtrsm' );
+```
+
+#### dtrsm( order, side, uplo, transa, diag, m, n, alpha, A, LDA, B, LDB )
+
+Solves matrix equation `op(A) * X = alpha * B` or `X * op(A) = alpha * B` where `alpha` is a scalar, `X` and `B` are m by n matrices, `A` is a unit, or non-unit, upper or lower triangular matrix and `op( A )` is one of `op( A ) = A` or `op( A ) = A ** T`. The matrix `X` is overwritten on `B`.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+A = new Float64Array( [ 1.0, 3.0, 0.0, 4.0 ] );
+B = new Float64Array( [ 5.0, 7.0, 0.0, 8.0 ] );
+
+dtrsm( 'row-major', 'left', 'upper', 'none', 'non-unit', 2, 2, 6.0, A, 2, B, 2 );
+// B => <Float64Array>[ 30.0, 6.0, 0.0, 12.0 ]
+```
+
+The function has the following parameters:
+
+-   **order**: storage layout of `A` and `B`.
+-   **side**: specifies whether `op( A )` appears on the left or right side of `X`.
+-   **uplo**: specifies whether the upper or lower triangular part of the matrix `A` is supplied.
+-   **transa**: specifies the form of `op( A )` to be used in the matrix multiplication.
+-   **diag**: specifies whether or not `A` is unit triangular.
+-   **m**: number of rows in `B`.
+-   **n**: number of columns in `B`.
+-   **alpha**: scalar constant.
+-   **A**: input matrix `A`.
+-   **LDA**: stride of the first dimension of `A` (a.k.a., leading dimension of the matrix `A`).
+-   **B**: input matrix `B`.
+-   **LDB**: stride of the first dimension of `B` (a.k.a., leading dimension of the matrix `B`).
+
+```javascript
```

Comment:
This example has no lead in and appears to be a duplicate of the first example. I suggest removing.